package.json

```diff
@@ -805,6 +805,7 @@
     "JSONStream": "1.3.5",
     "adm-zip": "^0.5.9",
     "ajv": "^8.12.0",
+    "ansi-regex": "^5.0.1",
```

Comment:
__note:__ would you mind assigning this dep to a Core team in `rennovate.json`? :slightly_smiling_face:

Reply:
sure! on it!

Reply:
Done in [`cde7a6e` (#164337)](https://github.com/elastic/kibana/pull/164337/commits/cde7a6e9a1bcc3837e603ad4d6cb7ba39e05d980)